docs/contributor-tutorial/.template/results.go

```diff
@@ -37,19 +37,19 @@ type UpdateResult struct {
 	commonResult
 }
 
-// ResourcePage is a single page of RESOURCE results.
-type ResourcePage struct {
+// SinglePage is a single page of RESOURCE results.
+type SinglePage struct {
```

Comment:
Seems this is just a example code, and actual implementation of each results.go is like:

```
package servers

type ServerPage struct {
	pagination.LinkedPageBase
}
```

so I would add "s" at the end of line 1: `package RESOURCEs` if the linter annoying us:)

Reply:
Thank you for your appropriate comments. I revised them as you suggested.

Reply:
LGTM!